stdpopsim/catalog/homo_sapiens.py

```diff
@@ -836,9 +836,9 @@ def __init__(self):
         N_Nean = 86.9
         N_LBK = 75.7
         N_Sard = 15000
-        N_Sard_LBK = 1200
-        # Nea pop size change 45,000 years ago (Jack Kamm pers comm)
-        t_NeaPopSizeChange = 45000 / self.generation_time
+        N_Sard_LBK = 12000
+        # Table A.1 has Altai at 50,000 years ago
+        t_NeaPopSizeChange = 50000 / self.generation_time
```

Comment:
Why is this different from the number given in the personal communication?

Reply:
That was my mistake - in the issue #167 I wrote Altai sampling of 45k, but I meant to write 50k (so I think Fernando was just following my typo). The Altai sampling was reported as 50k in the table in the ms.